src/HDF5.jl

```diff
@@ -2230,6 +2287,25 @@ function h5l_get_info(link_loc_id::Hid, link_name::String, lapl_id::Hid)
     h5l_get_info(link_loc_id, link_name, info, lapl_id)
     info[]
 end
+
+"""
```

Comment:
Include the call signature and some info on how to use it, or where to lookup how to use it? If these aren't exported you could turn them into comments.

Reply:
Turned them to comments.